usefulUtilities/sit/sitClient.js

```diff
@@ -446,57 +439,66 @@
         canClickOnModelToSit: false
     };
     function updateUserData() {
-        var properties = Entities.getEntityProperties(_this.entityID);
-        try {
-            _this.userData = JSON.parse(properties.userData);
-        } catch (e) {
-            console.log("Issue parsing userData" + e);
-        }
+        var userData = Entities.getEntityProperties(_this.entityID).userData;
```

Comment:
One more place where we can limit the properties requested

Reply:
ACK